Doc/Tutorial/chapter_align.tex

```diff
@@ -833,14 +833,14 @@ \subsection{Slicing alignments}
 
 \subsection{Alignments as arrays}
 Depending on what you are doing, it can be more useful to turn the alignment
-object into an array of letters -- and you can do this with NumPy:
+object into an array of letters (length one strings) -- and you can do this with NumPy:
```

Comment:
They are not strings but bytes. Accepted values are ASCII only. Compare to #3333 where we are also using bytes.

Reply:
You're right, https://numpy.org/doc/stable/reference/arrays.dtypes.html says "S" (and "a") are "zero-terminated bytes (not recommended)"

Should I leave the text as it was, or add something like "(length one byte sequence)"?

Reply:
I would suggest to use `"B"` instead. There is a reason by `"S"` is not recommended.

Reply:
Using bytes here gets more fiddly, e.g. 

```python
align_array = np.array([[ord(_) for _ in rec] for rec in alignment], "B")
```

or:

```python
align_array = np.array([list(rec.seq.encode()) for rec in alignment], "B")
```

Any preference, or a better way of writing this?

Reply:
Maybe there is value in showing both an array of single letter unicode strings (e.g. `A`), and an array of bytes (integer like `65`). It depends on why the user wants an array...

Reply:
i.e. using this:

```python
align_array = np.array([list(rec) for rec in alignment], "U")
```

Reply:
Or just remove this section from the documentation. I don't know what it is useful for.

Reply:
It was originally a user request and method suggestion for the multiple sequence alignment object.

If you have an array of characters, both row and column wise operations are trivially fast. Column operations on the current multiple sequence alignment object are slow.

I think I'll change it to the "U" version, seems closest to the original example.

Reply:
`"U"` ???

Reply:
"U" for unicode string, as per https://numpy.org/doc/stable/reference/arrays.dtypes.html#specifying-and-constructing-data-types

Reply:
But biological sequences don't contain unicode characters.

Reply:
I don't get your point here. Biological sequences are a sequences of residues or base pairs, each of which humans conventionally describe with words or single letters - and Python 3's unicode strings or ASCII bytes both work well as an abstraction.